content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
 
-[YouTube](https://youtu.be/0l_jrk0HTVY)
+Following this his team conducted live interviews with the selection of people using cognitive science techniques which enabled them to capture consistent data between different individuals across different religions, traditions and experiences that could be compared. This lead to the range of criteria of Fundamental Wellbeing, which has been continually researched by Dr. Martin. You can learn more about this in episode one of our interviews.
 
-[Spotify](https://spotifyanchor-web.app.link/e/c34gB9Bb8yb)
+## How do you reach Fundamental Wellbeing?
 
-Part 3: Dr Jeffery Martin on A Scientific Approach to Awakening and Fundamental Wellbeing 
+Those who have transitioned to Fundamental Wellbeing have done so through many gateways and environments, from deep spiritual devotion to psychedelic experiences. However, through his research Dr. Martin has developed a protocol for people to follow to transition to Fundamental Wellbeing.
 
-[Lifeitself Podcast](https://lifeitself.org/podcast/a-scientific-approach-to-awakening-and-fundamental-wellbeing-part-3-jeffery-martin)
+Learn more about is this Part 2 of the interviews.
+## What is the Martin Matrix? 
 
-[YouTube](https://youtu.be/vDuh7vDgGIg)
+Based on Dr. Martin's research, he was able to create a matrix/map of Fundamental Wellbeing and describe the different dimensions of experience. The axis are the location and the layer, where location describes the level of Fundamental Wellbeing and the layer describes the depth of the experience. 
 
-[Spotify](https://podcasters.spotify.com/pod/dashboard/episode/e22vp6s)
+The deepest layer and highest location is the peak stat of Fundamental Wellbeing. Learn more about the Martin Matrix and the descriptions of the layers and locations in part 2 and 3 of the interview and in this video below. 
 
-## Key readings and resources
+<iframe width="560" height="315" src="https://www.youtube.com/embed/aCfeamM07dk?si=mfdG1djlUneojhGf" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+## Who is Dr. Jeffery Martin?
+
+Jeffery Martin is a renowned author, speaker, and entrepreneur who has dedicated his life to studying and helping people achieve transformative experiences. He is the founder of the Transformative Technology Lab, a non-profit organization that seeks to develop and promote technologies that enhance human well-being.
```

Comment:
This section (next 3 paras) should have a heading like "About Jeffery Martin" (or even should just be deleted).